tests/test_dxl_single_chain.py

```diff
@@ -24,31 +30,39 @@
     ports = ['/dev/ttyUSB0']
 
 
-dxl_num = int(input("Enter a number to choose dynamixel type. 1 for MX-106, 2 for DXLPRO"))
-if dxl_num == 1:
-    dxl_str = "MX106"
-elif dxl_num == 2:
-    dxl_str = "DXLPRO"
-else:
-    print("Please a number between 1 and 2.")
-    exit(1)
-print(dxl_str)
+#dxl_num = int(input("Enter a number to choose dynamixel type. 1 for MX-106, 2 for DXLPRO"))
+#if dxl_num == 1:
+#    dxl_str = "MX106"
+#elif dxl_num == 2:
+#    dxl_str = "DXLPRO"
+#else:
+#    print("Please a number between 1 and 2.")
+#    exit(1)
+#print(dxl_str)
+
+dxl_str = "DXLPRO"
 
-dxl_opts = DxlOptions([[1,2]],
+dxl_opts = DxlOptions([[11]],
```

Comment:
we should probably keep this being a chain of two or more dynamixels so that we don't have issues with not handling multiple DXLs correctly. It looks like this is a script for the torque experiment, maybe separate stuff for that into its own file and then use `git checkout` to revert this file back to its original state?